test/backend/fixtures/ct-2.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+    "id": 1,
+    "submitter": "Tester",
+    "created_at": "2019-11-07
```

Comment:
@TheDanieLNK Is this JSON file intentionally invalid? It appears that the file got cut off.

Reply:
Yes. Just a test for malformed results.